pci/graphic_drivers/xf86-video-amdgpu/MHWDCONFIG

```diff
@@ -11,54 +11,24 @@ VENDORIDS="1002"
 DEVICEIDS=">/var/lib/mhwd/ids/pci/amdgpu.ids"
 
 # Dependencies
-DEPENDS="xf86-video-amdgpu"
-
+DEPENDS="xf86-video-amdgpu vulkan-radeon libva-mesa-driver libva-vdpau-driver mesa-vdpau"
+DEPENDS_64="lib32-vulkan-radeon lib32-libva-vdpau-driver lib32-mesa-vdpau"
 
+# Files
 XORGFILE="/etc/X11/mhwd.d/amdgpu.conf"
-MODULESFILE="/etc/modules-load.d/mhwd-amdgpu.conf"
-MODULESBLACKLIST="/etc/modprobe.d/mhwd-amdgpu.conf"
 
 post_install()
 {
-	MHWD_HEADING "${XORGFILE}"
-	MHWD_ADD_DEVICE_SECTION_FOR_EACH_BUSID "amdgpu" "${XORGFILE}" 1002
-	MHWD_ADD_DRI "${XORGFILE}"
-	MHWD_ADD_COMPOSITING "${XORGFILE}"
-	MHWD_ADD_BACKSPACE "${XORGFILE}"
-
```

Comment:
Yep. we dont need a xorg file for all the free drivers as long as we do not want to change the default. I just keep the xorgfile path for the removing. thats just how the other smaller drivers
https://github.com/manjaro/mhwd-db/blob/master/pci/graphic_drivers/xf86-video-ark/MHWDCONFIG
do this)